src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatabilityAnalyzerTests.cs

```diff
@@ -180,6 +181,48 @@ public int M3(int option)
             await VerifyAnalyzerAsyncCs(source);
         }
 
+        [Fact, WorkItem(4071, "https://github.com/dotnet/roslyn-analyzers/issues/4071")]
+        public async Task OsDependentPropertyGetterSetterCalledWarns()
+        {
+            var source = @"
+using System.Runtime.Versioning;
+
+public class Test
+{
+    public static bool WindowsOnlyPropertyGetter
+    {
+        [SupportedOSPlatform(""windows"")]
+        get { return true; }
+        set { }
+    }
+
+    public static bool WindowsOnlyPropertySetter
+    {
+        get { return true; }
+        [SupportedOSPlatform(""windows"")]
+        set { }
+    }
+
+    public void M1()
+    {
+        WindowsOnlyPropertyGetter = true;
+        var s = [|WindowsOnlyPropertyGetter|];
+        [|WindowsOnlyPropertyGetter|] |= true;
+        [|WindowsOnlyPropertySetter|] &= false;
+        [|WindowsOnlyPropertySetter|] = false;
+        s = WindowsOnlyPropertySetter;
+        M2([|WindowsOnlyPropertyGetter|]);
+        var name = nameof([|WindowsOnlyPropertyGetter|]);
```

Comment:
This doesn’t really invoke the getter. Maybe we should use the property symbol itself for this case?

Reply:
Well the property `WindowsOnlyPropertyGetter` has attribute only on the getter, not on the property itself, so as it is warning here i would assume it is invoking getter, no?

Reply:
Actually you are right, it's warning here because i make it 😄, will update